common/lib/xmodule/xmodule/course_module.py

```diff
@@ -555,15 +556,15 @@ class CourseFields:  # lint-amnesty, pylint: disable=missing-class-docstring
     certificates_display_behavior = String(
         display_name=_("Certificates Display Behavior"),
         help=_(
-            "Enter end, early_with_info, or early_no_info. After certificate generation, students who passed see a "
+            "Enter end, end_with_date, or early_no_info. After certificate generation, students who passed see a "
             "link to their certificates on the dashboard and students who did not pass see information about the "
```

Comment:
Is this still true after the changes? Does `early_no_info` show the grade info? I was under some assumption that `early_with_info` shows the grade date on the dashboard.

Reply:
So... this text doesn't actually get shown anywhere now that the field is not longer in Advanced settings. I'm not debating removing most of the historical language and shrinking it.

Reply:
done :)